### PR TITLE
Added props to show a infinite list effect to the slot

### PR DESCRIPTION
**Pull request by blzpedro**
These changes have already worked for me. 

E.g.: if you add some padding or increase the height from the char elements you wouldn't have the infinite dummy list effect. Adding the prop hasInfiniteList={true}, you will see the behavior correctly.

Feel free to change or improve!

Thanks

### CODE REVIEW

File: src/components/Slot.tsx [hunk 1]
```diff
@@ -57,6 +59,14 @@ function Slot({
...
   }, [value, dummyList]);
 
+  const renderDummyList = () => {
+    return dummyListState.map((dummyNumber, slotIndex) => (
+      <div key={slotIndex} className={styles.num} aria-hidden="true">
+        {dummyNumber}
+      </div>
+    ))
+  }
+  console.log(hasInfiniteList)
```
Comment (almond-bongbong) [inline src/components/Slot.tsx:69]: Please just remove the console.log, and I'll immediately merge it. Thank you!

**Comment by almond-bongbong** [comment]
What do you think about defining the hasInfiniteList option as the default behavior instead of providing it as an option?

**Comment by blzpedro** [comment]
I agree, and actually, I expected this as the default behavior when I got the package. 😂

But I think the changes that I made wouldn't work for all the cases that you provided. 

E.g: the slots that are used with any separator.

File: src/components/Slot.tsx (hunk 1 above)
Comment (blzpedro) [reply src/components/Slot.tsx]: Removed!

**Review by almond-bongbong — APPROVED** [review]
Thank you for your contribution!

**Comment by blzpedro** [comment]
I've thought of a solution for this edge case with values containing separators. We can avoid a **if (SEPARATOR.includes(v))** and threat this directly into the dummyList prop. What do you think?

Note: this would change the previous behavior where you had a separatorClassName. Maybe you can improve this?

```          
<Slot
    key={valueRefList.length - i - 1}
    fontHeight={fontHeight}
    numbersRef={numbersRef}
    active={active}
    isChanged={isChanged}
    charClassName={charClassName}
    slotNumbersHeight={slotNumbersHeight}
    effectiveDuration={effectiveDuration}
    delay={delay}
    value={v}
    dummyList={SEPARATOR.includes(v) ? dummyList.map(() => v) : dummyList}
    hasInfiniteList={hasInfiniteList}
/>
 ```

**Comment by almond-bongbong** [comment]
> How about adding an "animateSeparator" option? 😆

**Comment by blzpedro** [comment]
Yeah, it can be, to improve this you can add the animateSeparator depending on the hasInfiniteList to avoid the following behavior if you increase the height or add some paddingY to the chars(with the prop hasInfiniteList as true), the separator will not follow the centralized value.

**Pull request merged**